stripe/src/test/java/com/stripe/android/paymentsheet/PaymentOptionsActivityTest.kt

```diff
@@ -17,6 +17,7 @@ import com.stripe.android.model.PaymentMethodFixtures
 import com.stripe.android.paymentsheet.PaymentOptionsViewModel.TransitionTarget
 import com.stripe.android.paymentsheet.analytics.EventReporter
 import com.stripe.android.paymentsheet.analytics.SessionId
+import com.stripe.android.paymentsheet.model.FragmentConfigFixtures
```

Comment:
I am not sure what we want to choose to do here, but it seems like PaymentSheetActivityTest is used to test all the stuff in BaseSheetActivity, so the tests don't need to be duplicated in both test classes.

Reply:
Done.